src/Views/Game.hs

```diff
@@ -120,10 +120,13 @@ addGame api gameID = do
       appendChild gameListDiv gameDiv
 
       clickEventString gameName $ do
-        onServer $ joinGame api <.> gameID
-        players <- onServer $ findPlayersInGame api
-        deleteLobbyDOM
-        createGameDOM api
+        bool <- onServer $ joinGame api <.> gameID
+        case bool of
+          False -> return ()
```

Comment:
Better error handling would be preferred. Notice the user that he wasn't able to join the game.
We also need to find some way to tell the user why he couldn't join but that information isn't available here at the moment, so skip that and maybe create an issue for it?